crates/rpc/rpc/src/eth/api/traits/transaction.rs

```diff
@@ -125,24 +125,6 @@ pub trait EthTransactions: Send + Sync {
     /// Returns default gas limit to use for `eth_call` and tracing RPC methods.
     fn call_gas_limit(&self) -> u64;
 
-    /// Executes the future on a new blocking task.
-    ///
-    /// Note: This is expected for futures that are dominated by blocking IO operations, for tracing
-    /// or CPU bound operations in general use [Self::spawn_blocking].
-    async fn spawn_blocking_future<F, R>(&self, c: F) -> EthResult<R>
-    where
-        F: Future<Output = EthResult<R>> + Send + 'static,
-        R: Send + 'static;
-
-    /// Executes a blocking on the tracing pol.
-    ///
-    /// Note: This is expected for futures that are predominantly CPU bound, for blocking IO futures
-    /// use [Self::spawn_blocking_future].
```

Comment:
we no longer have this differentiation because we now always spawn all tasks on the CPU bound pool